apps/ui/src/components/ProposalLabels.vue

```diff
@@ -0,0 +1,131 @@
+<script setup lang="ts">
+import { Draft, Proposal, SpaceMetadataLabel } from '@/types';
+
+const props = withDefaults(
+  defineProps<{
+    labels: SpaceMetadataLabel[];
+    proposal: Proposal | Draft;
+    showEdit?: boolean;
+    inline?: boolean;
+  }>(),
+  {
+    showEdit: false,
+    inline: false
+  }
+);
+
+const emit = defineEmits<{
+  (e: 'updateProposalLabels', id: string): void;
+}>();
+
+const searchValue = ref('');
+const labelSelectorOpen = ref(false);
+const labelSelector = ref(null);
+
+const selectedLabels = computed(() => {
+  return props.proposal.labels
+    .filter(id => props.labels.some(label => label.id === id))
+    .map(id => {
+      return props.labels.find(label => label.id === id)!;
+    });
+});
+
+const filteredLabels = computed(() =>
+  props.labels.filter(label => {
+    return (
+      label.name.toLowerCase().includes(searchValue.value.toLowerCase()) ||
+      label.description.toLowerCase().includes(searchValue.value.toLowerCase())
+    );
+  })
+);
+
+onClickOutside(labelSelector, () => {
+  if (labelSelectorOpen.value) {
+    labelSelectorOpen.value = false;
+  }
+});
+
+function handleLabelClick(id: string) {
+  emit('updateProposalLabels', id);
+}
+</script>
+<template>
+  <div v-if="inline" class="contents">
+    <div v-for="label in selectedLabels" :key="label.id" class="mr-2">
+      <UiProposalLabel :label="label.name" :color="label.color" />
+    </div>
+  </div>
+  <div v-else>
+    <div class="flex justify-between">
+      <h4 class="eyebrow mb-2" v-text="'Labels'" />
+      <button
+        v-if="showEdit"
+        type="button"
+        class="h-0"
+        @click="labelSelectorOpen = true"
+      >
+        <IH-pencil />
+      </button>
+    </div>
+    <div
```

Comment:
Is it possible to use `Combobox` here ?

Reply:
Need some refactoring/add many conditions to handle the new design. don't think it worth, wyt @Sekhmet 

Combobox:
![image](https://github.com/user-attachments/assets/0f0d30c0-43a4-4f72-a409-0eb08025e458)
LabelSelctor
![image](https://github.com/user-attachments/assets/9142e476-d983-4b5c-9e65-6d7158026b3f)

Reply:
Looks like for accessbility, i will look into it

Reply:
I wouldn't use our Combobox directly, but rather build it using raw Combobox from headless UI:
https://headlessui.com/v1/vue/combobox